src/segment_media.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+)
+
+const (
+	// MediaPlayingIcon indicates a song is playing
+	MediaPlayingIcon Property = "playing_icon"
+	// MediaPausedIcon indicates a song is paused
+	MediaPausedIcon Property = "paused_icon"
+	// MediaStoppedIcon indicates a song is stopped
+	MediaStoppedIcon Property = "stopped_icon"
+	// MediaTrackSeparator is put between the artist and the track
+	MediaTrackSeparator Property = "track_separator"
+	// MediaTimeSeparator is put between the media position and total time
+	MediaTimeSeparator Property = "time_separator"
+	// MediaShowTime is media time show or hidden switch
+	MediaIsShowTime Property = "is_show_time"
```

Comment:
Our other `displaying` switches are prepended with `display_` so I would rename this to `display_time`.